_posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Rename the file as 2020-7-04-drsrija-intro-rfkill.md

Reply:
Done